README.md

```diff
@@ -1,34 +1,26 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
+**Before you initiate a pull request**, please read the process document. Ideas should be thoroughly discussed on the [Ethereum Magicians forums](https://ethereum-magicians.org) first.
```

Comment:
I like Ethereum magicians, but I know some people don't like it.  In the past, Gitter and GitHub issues have also been recommended for discussion prior to an EIP.  Any reason not to include them here?

Reply:
I myself don't like Ethereum Magicians. Only because email notifications aren't working.

But most importantly, we should recommend a single place where people can go to have in depth discussions about standards and improvements. And GitHub is not a forum so GitHub issues is probably not the correct place.

To be consistent, this issue also directly the repository owner to disable the Github Issues feature for this repository. Such is consistent with how Apple's Swift Evolution repository is governed. And I have borrowed ideas heavily from them -- including the two-week EIP review.